nes_bnf_komentarz.md

```diff
@@ -0,0 +1,26 @@
+<h2>
```

Comment:
Jeśli to jest `.md` to powinno używać się `#` etc. - nie uważam, że szkoda czasu na przeróbki.